Cargo.toml

```diff
@@ -63,4 +67,4 @@ unstable = []
 dev = []
 
 syntax-highlight = ["syntect"]
-
+sass = ['sass-rs']
```

Comment:
Please update .travis.yml
- `cargo check` for various feature combinations
- `cargo test` and clippy should run with all features enabled

If we feel we should have sass be included by default in our pre-built binaries, then we should also mention this features in [`before_deploy`](https://github.com/cobalt-org/cobalt.rs/blob/master/ci/before_deploy.sh#L16).

I'd recommend appveyor also being updated but I'm unsure if there is a good way for us to conditionally enable for this (because of it being unsupported for gnu).

Reply:
Done and done. And travis is passing, [but clippy complains on nightly, though I think that it is a false positive](https://travis-ci.org/cobalt-org/cobalt.rs/jobs/282053850#L2178-L2223).